feature-utils/poly-look/test/react-components/cardList.test.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import { fireEvent, render } from "@testing-library/react";
+import { Card, CardList } from "../../src/react-components";
+import "@testing-library/jest-dom";
+import { HistoryStub } from "../utils/history-stub";
+import { INITIAL_HISTORY_STATE } from "../../src/constants/state";
+
+const testH1 = "TestH1";
+
+const singleBasicCard = (
+  <Card>
+    <h1>{testH1}</h1>
+  </Card>
+);
+
+//testing navigation
+let clickedCard;
+const history = new HistoryStub();
+const testRoute = "/testRoute";
+const testStateChange = { scrollingProgress: 200 };
```

Comment:
Just to be on the safe side, shouldn't you first merge with `INITIAL_HISTORY_STATE` and then check that this is not the same value? You're assuming that it will always be 0, but things change :-)

Reply:
But INITIAL_HISTORY_STATE could potentially have other attributes. Also, for all we know, `scrollingProgress` could be 200. Baseline is, maybe consider assigning INITIAL_HISTORY_STATE to testStateChange and then change `scrollingProgress` by a known amount?

Reply:
Ah i get it: stateChange and state are different things. I think the issue you mean is solved by this here:
```
  const handleDivClick = () => {
    onClick && onClick();
    route && history.push(route, { ...INITIAL_HISTORY_STATE, ...stateChange });
  };
``` There `INITIAL_HISTORY_STATE` is overwritten by the changes